poll/tests.py

```diff
@@ -1,3 +1,75 @@
-# from django.test import TestCase
+import pytest
+from .models import Poll, get_default_end_date
+from datetime import datetime, timedelta
+from event.models import Event
+from category.models import Category
+from location.models import Location
+from category_location.models import CategoryLocation
 
-# Create your tests here.
+EVENT_NAME = "weight lifting"
+CATEGORY = "Gym"
+LOCATION = "Holmes place Netanya"
+MAX_PART = 10
+IS_PRIVATE = False
+DATETIME = datetime.now()
+
+
+@pytest.mark.django_db()
```

Comment:
Now I think you can replace the django_db with:
```suggestion
@pytest.fixture
```